### DISCUSSION TITLE
How to connect  Java spring boot backend &  MySql database

### DISCUSSION THREAD

**Discussion by Pudiing** [supabase/supabase#6715] [Questions]
I have two questions
How could I connect my Java Spring Boot backend with the application on teta
How can I use my MySql database instead of supabase or can I connect my MySql databe on supabase?

**Comment by dshukertjr** [comment]
Hi @Pudiing,

>How could I connect my Java Spring Boot backend with the application on teta

I have pinged the folks at teta about this question, so I'm going to let them answer this one 😉

>How can I use my MySql database instead of supabase or can I connect my MySql databe on supabase?

Again, whether teta is compatible with MySQL or not might be better answered by someone from teta, but I will take the second part of this question. With Supabase, PostgreSQL is the only database you can connect to, so no you cannot connect your MySQL database to Supabase.

**Comment by Pudiing** [comment]
Okey , thanks for answer, another question
What backend do u gays support! 😀

Dana nedjelja, 8. svibnja 2022. Tyler ***@***.***> piše:

**Reply by dshukertjr** [reply]
You are working on  creating a Flutter application, correct? The great things about Supabase is that you can build a secure and fully functional app without any backend code except for situations like credit card processing. For tasks like that, you can use [edge functions](https://supabase.com/edge-functions). Edge functions can be written in TypeScript. 

If for some reason edge functions does not fit your needs, you can brows the list of client libraries that we have [here](https://supabase.com/docs/reference/javascript/supabase-client). 

Again though, I would first think about whether you really need a backend. Supabase is a solution that manages your backend database security/logic for you, so you should be able to build your app with little to no backend code!
